docs/en/quick-answers/administering/share-assets-between-projects.md

```diff
@@ -9,20 +9,38 @@ lang: en
 
 It's not uncommon to have a project that's used as an Asset Library, containing assets that can be loaded into shots in other projects.
 
-To achieve this, you can add a tab to the [Loader app](https://support.shotgunsoftware.com/hc/en-us/articles/219033078) that lists assets from this Asset Library project. In order to do so, you'll have to define this in the Loader settings for the engine and environment you're working in. You'll potentially have to update this in multiple places.
+With the introduction of the Linked Projects Field on the Asset Entity, you can now add a single tab to the [Loader app](https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Supervisor_Artist_sa_integrations_sa_integrations_user_guide_html#the-loader) that includes all Linked Projects. In order to do so, you'll have to define this in the Loader settings for the engine and environment you're working in. You'll potentially have to update this in multiple places. 
```

Comment:
With the introduction of the Linked Projects field on the Asset entity, you can now add a single tab to the...